### ISSUE TITLE
Rotate filter applied too late compared to other image manipulation

### ISSUE THREAD

**Issue by smalot** [cshum/imagor#78]
I tried to use the rotate filter with a crop.
With a value of 180 degree, it works perfectly, but with a 90 degree rotation I obtained a squared image.

As you can see, rotation is correctly applied, but the crop with final dimensions is already applied, which results in generating a square.
It seems rotation is applied too late.

Here is the original image I used : http://coyote.vtt.free.fr/vw/MireCouleur16-9.jpg

180 degrees rotation:
> http://localhost:8000/unsafe/125x772/filters:rotate(180):fill(white)/http://coyote.vtt.free.fr/vw/MireCouleur16-9.jpg

here is the result:
![180](https://user-images.githubusercontent.com/1424035/178029388-52c0ade4-2049-4030-af51-cf2393df7e46.jpg)

-----
90 degrees rotation:

> http://localhost:8000/unsafe/125x772/filters:rotate(90):fill(white)/http://coyote.vtt.free.fr/vw/MireCouleur16-9.jpg

here is the result:
![90](https://user-images.githubusercontent.com/1424035/178029383-2a59055f-c661-4303-9a4e-a67b0d979cf2.jpg)

**Comment by smalot** [comment]
Hi @cshum 
Can you look at this issue if you have some time ?
Don't hesitate if you need more details about this use case.
Many thanks in advance

**Comment by cshum** [comment]
It seems the `fill()` filter has conflict with the rotation with cropping, will need to investigate more. 

Meanwhile dropping `fill()` filter should produce the desired image?

```
http://localhost:8000/unsafe/125x772/filters:rotate(90)/http://coyote.vtt.free.fr/vw/MireCouleur16-9.jpg
```
![MireCouleur16-9](https://user-images.githubusercontent.com/293790/178406796-a0fa9788-e508-4f15-9022-b051ac249b46.jpeg)

**Comment by cshum** [comment]
now fixed in v0.9.8:
https://github.com/cshum/imagor/pkgs/container/imagor/29395175?tag=v0.9.8

```
http://localhost:8000/unsafe/125x772/filters:rotate(90):fill(white)/http://coyote.vtt.free.fr/vw/MireCouleur16-9.jpg
```
![MireCouleur16-9](https://user-images.githubusercontent.com/293790/180673221-0e04aec9-4a42-452d-b4f6-3d7380f13911.jpeg)
